.github/workflows/access-control-integration-test.yml

```diff
@@ -83,7 +87,8 @@ jobs:
       - name: Authorization Integration Test (JDK${{ matrix.java-version }}-${{ matrix.test-mode }}-${{ matrix.backend }})
         id: integrationTest
         run: |
-          ./gradlew -PskipTests -PtestMode=${{ matrix.test-mode }} -PjdbcBackend=${{ matrix.backend }} -PjdkVersion=${{ matrix.java-version }} -PskipDockerTests=false :authorizations:authorization-ranger:test --tests "org.apache.gravitino.authorization.ranger.integration.test.**"
+          ./gradlew -PskipTests -PtestMode=embedded -PjdbcBackend=h2 -PjdkVersion=${{ matrix.java-version }} -PskipDockerTests=false :authorizations:authorization-ranger:test --tests "org.apache.gravitino.authorization.ranger.integration.test.**"
+          ./gradlew -PskipTests -PtestMode=deploy -PjdbcBackend=mysql -PjdkVersion=${{ matrix.java-version }} -PskipDockerTests=false :authorizations:authorization-ranger:test --tests "org.apache.gravitino.authorization.ranger.integration.test.**"
```

Comment:
@xunliu What's your opinion? Do you think it's necessary to run AC IT on both Mysql and PG backend in deploy mode?

Reply:
Yes, I think we can run AC IT on PG backend in the deploy mode.

Reply:
ok, added

Reply:
@mchades 
I think you can add a matrix parameter about `jdbcBackend` to reduce repeated line L92.

Reply:
That will produce extra CI jobs, so I put it in the step

Reply:
The current way will only generate one CI job, if it is written in matrix, it will generate three CI jobs, which will take up the concurrency of the CI.